x-pack/plugins/screenshotting/server/layouts/preserve_layout.css

```diff
@@ -1,93 +1,19 @@
-/*
- ******
- ****** This is a collection of CSS overrides that make Kibana look better for
- ****** generating PDF reports with headless browser
- ******
- */
-
 /**
- *  global
+ *  Global utilities
  */
 
 /* elements can hide themselves when shared */
 .hide-for-sharing {
```

Comment:
We can open up a second issue to follow up on this.

Reply:
I think what you wrote about just doing this in JS within the screenshotting plugin is a good idea